### ISSUE TITLE
Remedy.lic fails to get 'red flowers'

### ISSUE THREAD

**Issue by elBlanquito** [rpherbig/dr-scripts#1681]
```
You put your mortar in your oilcloth pack.
> 
--- Lich: remedy has exited.

[workorders]>get my alchemy logbook

You get an alchemy work order logbook from inside your oilcloth pack.
> 
[workorders]>bun my cream with logbook

You notate the cream in the logbook then bundle it up for delivery.
> 
[workorders]>stow my logbook

Cache thoroughly analyzes the genich ointment (external torso scars).
> 
You put your logbook in your oilcloth pack.
> 
--- Lich: remedy active.

[remedy]>get remed book

You get a book of apprentice remedy instructions from inside your oilcloth pack.
> 
[remedy]>turn my book to chapter 2

> 
You turn your book to chapter 2, entitled "Simple Remedies".
> 
[remedy]>read my book

              -=    Chapter 2 : Simple Remedies    =-

 Page 1: some blister cream (external head wounds)                      Page 4: some wart salve (external eye wounds)
 Page 2: some itch salve (external limb wounds)                         Page 5: some stomach tonic (internal abdomen wounds)
 Page 3: some moisturizing ointment (external chest wounds)              

> 
[remedy]>turn my book to page 1

You turn your book to page 1, instructions for crafting some blister cream (external head wounds).
> 
[remedy]>study my book

You review the blister cream (external head wounds) instructions and comprehend all but several minor details in the text. (4/5)
You now feel ready to begin the crafting process.
Roundtime: 8 sec.
R> 
[remedy]>put my book in my pack

You put your book in your oilcloth pack.
> 
[remedy]>get my mortar

You get a black marble mortar speckled with tiny silver veins from inside your oilcloth pack.
> 
[remedy]>get my red flowers

What were you referring to?
> 
[remedy: You seem to be missing: red flowers]

--- Lich: remedy has exited.

[DRPrime]-DR:Uidin: "Depends on what client you're using"
[workorders]>get my alchemy logbook

You get an alchemy work order logbook from inside your oilcloth pack.
> 
[workorders]>bun my cream with logbook

That's not going to work.  Type BUNDLE HELP for details.
> [Trigger]: skills
You've gained a new rank in your knowledge of the outdoors.
> 
Circle: 5
Showing all skills with field experience.

          SKILL: Rank/Percent towards next rank/Amount learning/Mindstate Fraction
    Shield Usage:     21 79% rapt          (30/34)     Light Armor:     18 83% interested    (12/34)
     Chain Armor:     22 03% very rapt     (31/34)      Brigandine:     16 78% learning       (3/34)
     Plate Armor:     21 08% studious      (19/34)       Defending:     21 65% riveted       (28/34)
   Parry Ability:     20 14% very riveted  (29/34)     Small Edged:      8 10% perusing       (2/34)
   Melee Mastery:     14 80% interested    (12/34) Missile Mastery:     13 59% ruminating     (8/34)
         Evasion:     21 71% very riveted  (29/34)       Athletics:     24 88% fascinated    (25/34)
      Perception:     23 24% rapt          (30/34)         Stealth:     13 94% very rapt     (31/34)
 Outdoorsmanship:     17 02% pondering      (7/34)        Skinning:     12 96% intrigued     (16/34)
         Forging:     30 94% ruminating     (8/34)     Engineering:     26 38% studious      (19/34)
      Outfitting:     26 11% considering    (6/34)         Alchemy:     24 35% intrigued     (16/34)
     Scholarship:     20 70% dabbling       (1/34)         Tactics:     23 77% cogitating    (24/34)

Total Ranks Displayed: 433
Time Development Points: 8  Favors: 9  Deaths: 0  Departs: 0
Overall state of mind: clear
EXP HELP for more information

> 
Cache thoroughly analyzes the genich ointment (external torso scars).
> 
[DRPrime]-DR:Willower: "problem is, each spell has a different duration. And I'm stuck with SF for now"

[workorders: *** No match was found after 15 seconds, dumping info]

[workorders: messages seen length: 22]

[workorders: message: That's not going to work.  Type BUNDLE HELP for details.]

[workorders: message: You've gained a new rank in your knowledge of the outdoors.]

[workorders: message: Circle: 5]

[workorders: message: Showing all skills with field experience.]

[workorders: message:           SKILL: Rank/Percent towards next rank/Amount learning/Mindstate Fraction]

[workorders: message:     Shield Usage:     21 79% rapt          (30/34)     Light Armor:     18 83% interested    (12/34)]

[workorders: message:      Chain Armor:     22 03% very rapt     (31/34)      Brigandine:     16 78% learning       (3/34)]

[workorders: message:      Plate Armor:     21 08% studious      (19/34)       Defending:     21 65% riveted       (28/34)]

[workorders: message:    Parry Ability:     20 14% very riveted  (29/34)     Small Edged:      8 10% perusing       (2/34)]

[workorders: message:    Melee Mastery:     14 80% interested    (12/34) Missile Mastery:     13 59% ruminating     (8/34)]

[workorders: message:          Evasion:     21 71% very riveted  (29/34)       Athletics:     24 88% fascinated    (25/34)]

[workorders: message:       Perception:     23 24% rapt          (30/34)         Stealth:     13 94% very rapt     (31/34)]

[workorders: message:  Outdoorsmanship:     17 02% pondering      (7/34)        Skinning:     12 96% intrigued     (16/34)]

[workorders: message:          Forging:     30 94% ruminating     (8/34)     Engineering:     26 38% studious      (19/34)]

[workorders: message:       Outfitting:     26 11% considering    (6/34)         Alchemy:     24 35% intrigued     (16/34)]

[workorders: message:      Scholarship:     20 70% dabbling       (1/34)         Tactics:     23 77% cogitating    (24/34)]

[workorders: message: Total Ranks Displayed: 433]

[workorders: message: Time Development Points: 8  Favors: 9  Deaths: 0  Departs: 0]

> 
[workorders: message: Overall state of mind: clear]

[workorders: message: EXP HELP for more information]

[workorders: message: Cache thoroughly analyzes the genich ointment (external torso scars).]

[workorders: message: [DRPrime]-DR:Willower: "problem is, each spell has a different duration. And I'm stuck with SF for now"]

[workorders: checked against [/You notate/i, /You put/i, /You notice the workorder/i, /The work order requires items of a higher quality/i]]

[workorders]>stow right

Cache gets an iron pestle from inside his traveler's pack.
> 
You put your mortar in your oilcloth pack.
> 
[workorders]>stow left

You put your logbook in your oilcloth pack.
> 
[workorders]>get my alchemy logbook

You get an alchemy work order logbook from inside your oilcloth pack.
> 
[workorders]>give log to Lanshado

The work order isn't yet complete, so you decide not to bother Lanshado.
> 
[workorders]>put my logbook in my pack

You put your logbook in your oilcloth pack.
> 
--- Lich: go2 active.

[go2: ETA: 0:00:00 (2 rooms to move through)]

[go2]>north

With precise movements, Lanshado begins to crush some herbs to a fine dust using a kertig mortar and pestle.
> 
You go north.

[Alchemy Society, Bookstore]
A haze of dust permeates the air and casts a glow about the wall torches.  A large bookshelf rests against the back wall and a robed figure scurries up and down a ladder to sell tomes and recipes to interested patrons.
You also see a large sign.
Obvious exits: north, south.

Room Number: 8861

> 
[go2]>north

You go north.

[Alchemy Society, Tool Shop]
Tall oak supply crates flank an extended counter displaying various alchemical tools for sale.  A pair of robed individuals rush about helping customers restock.  You notice the overwhelming combination of smells is somewhat reduced here.
You also see a dry press, a sturdy low counter with a small placard on it, a large sign, a plant grinder and a clerk.
Obvious exits: north, south.

Room Number: 8860

> 
[go2: travel time: 0:00:01]

--- Lich: go2 has exited.

[Trigger]: skills
You've gained a new rank in your skinning ability.
> 
[workorders]>get my pestle

Circle: 5
Showing all skills with field experience.

          SKILL: Rank/Percent towards next rank/Amount learning/Mindstate Fraction
    Shield Usage:     21 79% rapt          (30/34)     Light Armor:     18 83% interested    (12/34)
     Chain Armor:     22 03% very rapt     (31/34)      Brigandine:     16 78% learning       (3/34)
     Plate Armor:     21 08% studious      (19/34)       Defending:     21 65% riveted       (28/34)
   Parry Ability:     20 14% very riveted  (29/34)     Small Edged:      8 10% perusing       (2/34)
   Melee Mastery:     14 80% interested    (12/34) Missile Mastery:     13 59% ruminating     (8/34)
         Evasion:     21 71% very riveted  (29/34)       Athletics:     24 88% fascinated    (25/34)
      Perception:     23 24% rapt          (30/34)         Stealth:     13 94% very rapt     (31/34)
 Outdoorsmanship:     17 02% pondering      (7/34)        Skinning:     13 04% absorbing     (15/34)
         Forging:     30 94% ruminating     (8/34)     Engineering:     26 38% studious      (19/34)
      Outfitting:     26 11% considering    (6/34)         Alchemy:     24 35% intrigued     (16/34)
     Scholarship:     20 70% dabbling       (1/34)         Tactics:     23 77% cogitating    (24/34)

Total Ranks Displayed: 434
Time Development Points: 8  Favors: 9  Deaths: 0  Departs: 0
Overall state of mind: clear
EXP HELP for more information

> 
You get a black marble pestle speckled with tiny silver veins from inside your oilcloth pack.
> 
[workorders]>give clerk

A clerk looks over the pestle and says, "That will cost 37 Kronars to repair.  Just give it to me again if you want, and I'll have it ready in 1 roisaen."
> 
[workorders]>give clerk

You hand the clerk 37 Kronars and he gives you back a repair ticket.  clerk says, "I should be having that done for you in about 1 roisaen.  Don't lose this ticket!  You must have it to reclaim your pestle."
> 
[workorders]>stow ticket

You put your ticket in your oilcloth pack.
> 
[DRPrime]-DR:Jhien: "I use SF, thats not an issue - lich can take info from the Active Spell Window"

[workorders]>get my mortar

You get a black marble mortar speckled with tiny silver veins from inside your oilcloth pack.
> 
[workorders]>give clerk

A clerk looks over the mortar and says, "That will cost 37 Kronars to repair.  Just give it to me again if you want, and I'll have it ready in 1 roisaen."
> 
[workorders]>give clerk

You hand the clerk 37 Kronars and he gives you back a repair ticket.  clerk says, "I should be having that done for you in about 1 roisaen.  Don't lose this ticket!  You must have it to reclaim your mortar."
> 
[workorders]>stow ticket

You put your ticket in your oilcloth pack.
> 
[workorders]>get my sieve

You get a trapezoidal darkstone wire sieve from inside your oilcloth pack.
> 
[workorders]>give clerk

A clerk says, "There isn't a scratch on that, so there's nothing to repair."
> 
[workorders]>put my sieve in my pack
```

**Comment by rpherbig** [comment]
I think this is the same as #1499

**Comment by rpherbig** [comment]
A smaller log from when this happened to me today:
```
2017-07-25 15:36:22 -0400:--- Lich: remedy active.
2017-07-25 15:36:22 -0400:[remedy]>get remed book
2017-07-25 15:36:23 -0400:You get a book of apprentice remedy instructions from inside your miner's backpack.
2017-07-25 15:36:23 -0400:[remedy]>turn my book to chapter 2
2017-07-25 15:36:23 -0400:You turn your book to chapter 2, entitled "Simple Remedies".
2017-07-25 15:36:23 -0400:[remedy]>read my book
2017-07-25 15:36:23 -0400:              -=    Chapter 2 : Simple Remedies    =-
2017-07-25 15:36:23 -0400: Page 1: some blister cream                      Page 4: some wart salve
2017-07-25 15:36:23 -0400: Page 2: some itch salve                         Page 5: some stomach tonic
2017-07-25 15:36:23 -0400: Page 3: some moisturizing ointment              
2017-07-25 15:36:23 -0400:[remedy]>turn my book to page 1
2017-07-25 15:36:23 -0400:You turn your book to page 1, instructions for crafting some blister cream.
2017-07-25 15:36:24 -0400:[remedy]>study my book
2017-07-25 15:36:24 -0400:You scan the blister cream instructions with a glance and confidently discern most of the design's minutiae.
2017-07-25 15:36:24 -0400:You now feel ready to begin the crafting process.
2017-07-25 15:36:24 -0400:Roundtime: 6 sec.
2017-07-25 15:36:30 -0400:[remedy]>put my book in my backpack
2017-07-25 15:36:31 -0400:You put your book in your miner's backpack.
2017-07-25 15:36:31 -0400:[remedy]>get my mortar
2017-07-25 15:36:31 -0400:You get an iron mortar from inside your miner's backpack.
2017-07-25 15:36:31 -0400:[remedy]>get my red flowers
2017-07-25 15:36:31 -0400:What were you referring to?
2017-07-25 15:36:31 -0400:[remedy: You seem to be missing: red flowers]
2017-07-25 15:36:31 -0400:--- Lich: remedy has exited.
```

**Comment by Sarvatt** [comment]
would changing the recipe to red flower instead of red flowers help?

**Comment by Dartellum** [comment]
I have fixed it. Red flower is how you purchase but the noun is dried: dried red flowers. I am testing and should have the PR up soon.

**Comment by rpherbig** [comment]
Fixed in a5dfd841da50f688fef0f3d6d1bd029c5fbca40e
